package.json

```diff
@@ -8,9 +8,6 @@
     "http": "./empty-module.cjs.js",
     "https": "./empty-module.cjs.js"
   },
-  "engines": {
-    "node": ">=8.16.0"
```

Comment:
I don't think at this time people are using something lower than 8.16.0.
So I got rid of them. Any concern?

Reply:
Seems fine I guess

Reply:
With this requirement it was not possible to use it with something lower than 8.16.0 anyway.

With the change in Ship.js you'll need node 12 to contribute but users still only need at least node 8.16.0 to use the package, right ?

If that is the case it is ok (not a breaking change) but you should also update `.travis.yml` to make the build on node 12 but test it on node 8 (is that possible on Travis ?)

Reply:
Yes, you can specify multiple versions like so:

https://github.com/algolia/create-instantsearch-app/blob/d9f14869ac80e1d4b1cfa87b11d45d903c3e56d6/.travis.yml#L2-L5

Reply:
But does this mean Travis is going to iterate and run tests per version?
For tests, it's okay, but we need to run `yarn shipjs trigger` on Travis, too.
According to the syntax, I don't think it's possible to run `yarn shipjs trigger` only on node v12, while other tests run on multiple versions.

Any idea?

Reply:
You could still conditionally run a Bash command in the script, but that'd suck to put logic in there to trigger a release.

Reply:
Agree. If we move it over to CircleCI, then we can smoothly implement it because in CircleCI we can specify node version per job and we can run multiple jobs.

Reply:
You can use nvm on Travis to only run some specific commands on node v12: `nvm exec 12 yarn shipjs trigger`

Reply:
@yannickcr ahhh that's a good idea. I like it. I'll update this branch to try it out!

Reply:
It's not going to work. With 8.16.0 in .travis.yml, it cannot even install dependencies.
For tests, it only needs v8 but for release it needs v12. And both needs to install devDependencies. It's not something that cannot differentiate with yarn install --production.
Later I'll find some time to move it over to CircleCI rather than struggling with Travis any longer.